tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1084,6 +1084,27 @@ private void writeSummaryTable(
 
     private Map<String, String> getSortedNames(Organization org, Level desiredLevel) {
         Map<String, String> sortedNames = new TreeMap<>(CLDRConfig.getInstance().getCollator());
+        LocalesWithExplicitLevel cldrLocale =
+                new LocalesWithExplicitLevel(Organization.cldr, desiredLevel);
+
+        // A user in the Unaffiliated organization can access a list with all non-TC locales.
+        if (org == Organization.unaffiliated && desiredLevel == Level.BASIC) {
+            for (String localeID : cldrFactory.getAvailable()) {
+                final CLDRLocale loc = CLDRLocale.getInstance(localeID);
+                if (SubmissionLocales.isTcLocale(loc)) continue; // skip TC locales
```

Comment:
Once this is in place, 
1. the Priority Items should have all and only those locales that are in Organization.cldr - Organization.special.
2. This list should include all and only those locales that are not in the Priority items.

Reply:
In particular, if a locale has any errors (that are not duplicates of a parent locales errors), they should be listed in one of the two lists.

Reply:
I didn't modify the non DDL list. Is it already correct?

Reply:
The "normal" code is here

https://github.com/unicode-org/cldr/blob/e899b786b8003d9f7b984bf710beeffd940a7682/tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java#L1091

Reply:
> In particular, if a locale has any errors (that are not duplicates of a parent locales errors), they should be listed in one of the two lists.

Could we include this under the separate ticket about deduplicating errors? This particular PR has a more limited scope, that simply having a way to get DDL reports at all.

Reply:
Yes, the deduping can be separate.